smqtk_classifier/impls/classifier/classify_index_label_descriptor.py

```diff
@@ -3,10 +3,10 @@
 import numpy as np
 from smqtk_dataprovider import from_uri
 
-from smqtk_classifier.interfaces.classifier import Classifier
+from smqtk_classifier.interfaces.classify_descriptor import ClassifyDescriptor
 
 
-class IndexLabelClassifier (Classifier):
+class ClassifyIndexLabelDescriptor(ClassifyDescriptor):
```

Comment:
I missed earlier that there are there are other class re-namings. These are also, ideally, in need of compatibility shims that throw deprecation warnings so that users are not immediately broken.